java/code/src/com/redhat/rhn/frontend/action/configuration/sdc/FileListConfirmSetupAction.java

```diff
@@ -30,13 +31,14 @@
 import org.apache.struts.action.DynaActionForm;
 
 import java.util.List;
+import java.util.Set;
 
 
 /**
  * FileListConfirmSetupAction, for sdc configuration
  * @version $Rev$
  */
-public class FileListConfirmSetupAction extends BaseListAction {
+public abstract class FileListConfirmSetupAction extends BaseListAction implements MaintenanceWindowsAware {
```

Comment:
Note for the reviewer: I made the original class abstract and split the actual implementation into 2 concrete classes (above) since struts action using this class resulted in 2 different backend actions (diff or deploy).